sdk/communication/communication-phone-numbers/src/sipRoutingClient.ts

```diff
@@ -299,13 +302,49 @@ export class SipRoutingClient {
     );
```

Comment:
Feedback from Review:
We should also expose "Domains", so we should implement these methods:

    getDomain(domainName: string, options?: OperationOptions): Promise<SipDomain>;
    listDomains(options?: OperationOptions): PagedAsyncIterableIterator<SipDomain>;
    setDomain(domain: SipDomain, options?: OperationOptions): Promise<SipDomain>;
    setDomains(domains: SipDomain[], options?: OperationOptions): Promise<SipDomain[]>;
    
The internal logic is similar to those handling trunks.
Also, we need to expose SipDomain object that would be handled similarly as SipTrunk.

Reply:
Added domains contract.